src/SlackClientFactory.ts

```diff
@@ -156,7 +156,7 @@ export class SlackClientFactory {
                     // non-ideal way to detect calls to slack.
                     webLog.debug.bind(webLog);
                     if (!this.onRemoteCall) { return; }
-                    const match = /apiCall\('([\w\.]+)'\) start/.exec(msg[0]);
+                    const match = /apiCall\('([\w\.]+)'\) start/.exec(msg);
```

Comment:
what's this change?

Reply:
Message is a string, we were regexing against the first char

Reply:
Message is a string, we were regexing against the first char